nessus-agent/elx/init.sls

```diff
@@ -30,6 +30,75 @@ Print nessus-agent help:
         without linking the agent to the server.
 {%- endif %}
 
+{%- if (nessus.log_rotation_strategy) %}
+Add Log Rotation Strategy:
+  file.replace:
+    - name: '/opt/nessus_agent/var/nessus/log.json'
+    - pattern: '"file": "/opt/nessus_agent/var/nessus/logs/www_server.log"'
+    - repl: '"rotation_strategy": {{nessus.log_rotation_strategy}},\n\t\t\g<0>'
+    - watch:
+      - cmd: Pause For Log File
+{%- endif %}
+
+{%- if (nessus.log_rotation_time) %}
+Add Log Rotation Time:
+  file.replace:
+    - name: '/opt/nessus_agent/var/nessus/log.json'
```

Comment:
Wrap values in single quotes only if truly needed for yaml to interpret the string properly. It should not be required in most cases.

Reply:
Okay.  I'll test without the single quotes.  I'll remove is if it works fine.